src/views/partials/card.handlebars

```diff
@@ -0,0 +1,52 @@
+{{#each card}}
+<div class="card-modal" hidden>
+    <button class="close-button" id="closeModal">Close Modal</button>
+    <form id="updateCard" action="/board/card/{{boardId}}/{{_id}}" method="POST">
+        <h1>{{name}}</h1>
+        <h2>Members</h2>
+        {{#each members}}
+        {{#if assigned}}
+        <h3 class="assigned">{{initials}}</h3>
+        {{else}}
+        <h3 class="unassigned">{{initials}}</h3>
+        {{/if}}
+        {{/each}}
+        <h2>Labels</h2>
+        {{#each labels}}
+        <h3 style="background-color: {{color}}">{{text}}</h3>
+        {{/each}}
+        <h2>Description</h2>
+        <textarea id="description" name="description">{{description}}</textarea>
+        <h2>Move Card</h2>
+        <label>
+            To List
+            <select id="toList">
+                {{#each list}}
+                <option value="{{_id}}">{{listName}}</option>
+                {{/each}}
+            </select>
+        </label>
+        <label>
+            Position
+            <select id="position">
+                {{#each position}}
+                <option value="{{position}}">{{position}}</option>
+                {{/each}}
+            </select>
+        </label>
+        <label>
+            Due Date
+            <input type="date" id="dueDate" value="{{dueDate.date}}">
```

Comment:
type="date" does not do anything with the time, should have a second input for the time as well